wordpress/mysql/Dockerfile_LOCAL

```diff
@@ -0,0 +1,34 @@
+# =============================================================================
+# mysql:5.7
+#
+# Modification: WordPress MySQL error log Fail2Ban setup
+#
+# =============================================================================
+FROM mysql:5.7
+LABEL maintainer="Jacob Thompson <jacob.thompson@wiwasolvet.ca>"
+
+RUN apt-get update &&  \
+    apt-get install -y \
+    nano \
```

Comment:
verify these are properly being installed, it seems like the updated repo isn't ready on build, so in CKAN I have to run 'apt get update' and 'apt install nano' etc. on fresh builds of docker containers.

Reply:
maybe it's just the 'dash' in debian vs whatever each container's environment actually is...